models/userStatus.js

```diff
@@ -136,6 +136,27 @@ const updateUserStatus = async (userId, newStatusData) => {
   }
 };
 
+/**
+ * Sets the monthlyHours field of passed UserId to numberOfHours
+ *
+ * @param monthlyHours { integer }: numberOfHours
+ * @param userId { string }: User id
+ */
+
+const updateMonthlyHours = async (numberOfHours, userId) => {
+  const userStatusDocs = await userStatusModel.where("userId", "==", userId).limit(1).get();
+  const [userStatusDoc] = userStatusDocs.docs;
+  if (userStatusDoc) {
+    const docId = userStatusDoc.id;
+    return userStatusModel.doc(docId).update({
+      monthlyHours: {
+        committed: numberOfHours,
+      },
+    });
+  }
+  return {};
+};
```

Comment:
can't this be accommodated in the updateUserStatus function?

Reply:
no because parameter is coming from controllers/user.js